src/at-unix.c

```diff
@@ -128,6 +135,39 @@ struct at *at_alloc_unix(const char *devpath, speed_t baudrate)
     return (struct at *) priv;
 }
 
+void at_reconf_parity(struct at *at)
```

Comment:
I would leave serial settings for underlying OS/RTOS. In our case it's reasonable to change Nuttx configuration rather than injecting different settings here.